liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerXmlDocument.java

```diff
@@ -42,38 +45,84 @@
     
     private ServerXmlDocument() {    
     }
+
+    /**
+     * @param filePath
+     * @param fileName application file name
+     * @return true if the application with the file name is configured.
+     * @throws Exception
+     */
+    private static boolean isFoundWebAppWithFileName(String filePath, String fileName) throws Exception {
+        return isFoundAppConfigByTagAndFileName(filePath, "webApplication", fileName);
+    }
     
-    public static boolean isFoundTagName(String fileName, String tag) throws Exception {
+    /**
+     * @param filePath 
+     * @param tag specify element tag to parse
+     * @param fileName application file name
+     * @return true if the application with the file name within the specified element tag is configured. 
+     * @throws Exception
+     */
+    private static boolean isFoundAppConfigByTagAndFileName(String filePath, String tag, String fileName) throws Exception {
```

Comment:
Basically, those comments are added to give more details for similarly named methods.